sway/input/cursor.c

```diff
@@ -680,6 +692,8 @@ static void handle_tool_tip(struct wl_listener *listener, void *data) {
 
 		wlr_tablet_v2_tablet_tool_notify_up(sway_tool->tablet_v2_tool);
 	}
+
+	seatop_tablet_tool_tip(seat, sway_tool, event->time_msec, event->state);
```

Comment:
Shouldn't this be inside the condition above, right after `seatop_tablet_tool_motion`?

Reply:
Err, my bad.